plugins/core/jetbrains-community/src/software/aws/toolkits/jetbrains/core/credentials/ConfigFilesFacade.kt

```diff
@@ -197,6 +198,31 @@ class DefaultConfigFilesFacade(
         }
     }
 
+    override fun deleteSsoProfileScopesFromConfig(sessionName: String) {
```

Comment:
what if a user is depending on the session to have scopes outside of the extension?